language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/extensions/ballerina/packages/PackageMetadataResponse.java

```diff
@@ -47,4 +48,12 @@ public String getKind() {
     public void setKind(String kind) {
         this.kind = kind;
     }
+
+    public String getOrgName() {
```

Comment:
we can remove this function

Reply:
Remove this will cause spotbug failure. We use this to generate RPC response.